password_change_restrict/models/res_users.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# License and authorship information in:
+# __openerp__.py file at the root folder of this module.
+
+from openerp import api, exceptions, models, SUPERUSER_ID, _
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    SELF_WRITEABLE_FIELDS = ['signature', 'action_id', 'company_id', 'email',
+                             'name', 'image', 'image_medium', 'image_small',
+                             'lang', 'tz']
+
+    def _can_change_password(self):
+        group_ok = 'base.group_erp_manager'
+        user = self.env['res.users'].browse(self.env.uid)
```

Comment:
@hbrunn I am having a weird issue with that. I explain:
Given a user that does not belong to `group_ok = 'base.group_erp_manager'`:
Using `user = self.env.user`, `user.has_group(group_ok)` returns True (should not!)
But using `user = self.env['res.users'].browse(self.env.uid)`, `user.has_group(group_ok)` returns False (correct)

Any idea why?